pywr/parameters/_parameters.pyx

```diff
@@ -670,6 +670,92 @@ cdef class ScenarioMonthlyProfileParameter(Parameter):
 
 ScenarioMonthlyProfileParameter.register()
 
+cdef class ScenarioWeeklyProfileParameter(Parameter):
+    """
+    This parameter provides a repeating annual profile with a weekly resolution. A
+    different profile is returned for each member of a given scenario
+    
+    Parameters
+    ----------
+    scenario: Scenario
+        Scenario object over which different profiles should be provided.
+    values : iterable, array
+        length of 1st dimension should equal the number of members in the scenario object 
+        and the length of the second dimension should be 52
+
+    """
+    def __init__(self, model, Scenario scenario, values, *args, **kwargs):
+        super().__init__(model, *args, **kwargs)
+        if values.ndim != 2:
```

Comment:
I wonder if we should do `values = np.array(values)` first? This would allow you to init with a list of list for example. As it is you must init with a numpy array.